# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Fabian Affolter <fab@fedoraproject.org>, 2011.
#  <hpeters@redhat.com>, 2013.
#   <hpeters@redhat.com>, 2011-2012.
# Jiří Moskovčák <jmoskovc@redhat.com>, 2011.
# Laurin  <lineak@fedoraproject.org>, 2011.
# Mario Blättermann <mariobl@freenet.de>, 2011.
msgid ""
msgstr ""
"Project-Id-Version: gnome-abrt\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2012-10-18 17:05+0200\n"
"PO-Revision-Date: 2013-01-22 01:40+0000\n"
"Last-Translator: hpeters <hpeters@redhat.com>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../src/gnome-abrt.desktop.in.h:1 ../src/gnome_abrt/views.py:71
msgid "Automatic Bug Reporting Tool"
msgstr "Automatisches Bug-Reporting-Tool"

#: ../src/gnome-abrt.desktop.in.h:2
msgid "View and report application crashes"
msgstr "Applikationsabstürze ansehen und melden"

#: ../src/gnome_abrt/dbus_problems.py:51
msgid "Can't add receiver of signal '{0}'on DBus system path '{1}': {2}"
msgstr "Empfänger von Signal '{0}' auf DBus-Systempfad '{1}' kann nicht hinzugefügt werden: {2}"

#: ../src/gnome_abrt/dbus_problems.py:73
msgid "Can't connect to DBus system bus '{0}' path '{1}': {2}"
msgstr "Verbindung mit DBus-Systembus '{0}' auf Pfad '{1}' nicht möglich: {2}"

#: ../src/gnome_abrt/dbus_problems.py:78
msgid "Can't get interface '{0}' on path '{1}' in bus '{2}': {3}"
msgstr "Schnittstelle '{0}' auf Pfad '{1}' in Bus '{2}' kann nicht abgerufen werden: {3}"

#: ../src/gnome_abrt/dbus_problems.py:124
msgid "Can't get problem data from DBus service: {0!s}"
msgstr "Fehlerdaten können nicht von DBus-Dienst abgerufen werden: {0!s}"

#: ../src/gnome_abrt/dbus_problems.py:139
msgid "Can't get list of problems from DBus service: {0!s}"
msgstr "Fehlerliste kann nicht von DBus-Dienst abgerufen werden: {0!s}"

#: ../src/gnome_abrt/dbus_problems.py:152
msgid "Can't delete problem over DBus service: {0!s}"
msgstr "Fehler kann nicht über DBus-Dienst gelöscht werden: {0!s}"

#: ../src/gnome_abrt/directory_problems.py:185
msgid "Can't open directory: '{0}'"
msgstr "Verzeichnis kann  nicht geöffnet werden: '{0}'"

#: ../src/gnome_abrt/oops.glade.h:1
msgid "Delete"
msgstr "Löschen"

#: ../src/gnome_abrt/oops.glade.h:2
msgid "Delete selected problems"
msgstr "Ausgewählte Fehler löschen"

#: ../src/gnome_abrt/oops.glade.h:3
msgid "_Report"
msgstr "_Bericht"

#: ../src/gnome_abrt/oops.glade.h:4
msgid "Submits selected problem"
msgstr "Berichtet ausgewählten Fehler"

#: ../src/gnome_abrt/oops.glade.h:5
msgid "Detail"
msgstr "Detail"

#: ../src/gnome_abrt/oops.glade.h:6
msgid "_All problems"
msgstr "_Alle Fehler"

#: ../src/gnome_abrt/oops.glade.h:7
msgid "Show all known problems from all system users"
msgstr "Alle bekannten Fehler für alle Systembenutzer anzeigen"

#: ../src/gnome_abrt/oops.glade.h:8
msgid "Name"
msgstr "Name"

#: ../src/gnome_abrt/oops.glade.h:9
msgid "Version"
msgstr "Version"

#: ../src/gnome_abrt/oops.glade.h:10
msgid "Reported"
msgstr "Berichtet"

#: ../src/gnome_abrt/oops.glade.h:11
msgid "No oopses!"
msgstr "Keine Kernel-Abstürze!"

#: ../src/gnome_abrt/oops.glade.h:12
msgid "App"
msgstr "App"

#: ../src/gnome_abrt/oops.glade.h:13
msgid "Date"
msgstr "Datum"

#: ../src/gnome_abrt/problems.py:151
msgid "Can't delete problem '{0}': '{1}'"
msgstr "Fehler '{0}' kann nicht gelöscht werden: '{1}'"

#: ../src/gnome_abrt/problems.py:286
msgid "Problem '{0}' is already in the cache"
msgstr "Fehler '{0}' ist bereits im Cache"

#: ../src/gnome_abrt/problems.py:300
msgid "Not found in cache but deleted: {0}"
msgstr "Nicht im Cache gefunden sondern gelöscht: {0}"

#: ../src/gnome_abrt/problems.py:314
msgid "Can't process '{0}': {1}"
msgstr "'{0}' kann nicht verarbeitet werden: {1}"

#: ../src/gnome_abrt/problems.py:316
msgid "Source failed on processing of '{0}': {1}"
msgstr "Quelle bei Verarbeitung von '{0}' fehlgeschlagen: {1}"

#: ../src/gnome_abrt/tools.py:18
msgid "Future"
msgstr "Zukunft"

#: ../src/gnome_abrt/tools.py:25
msgid "Yesterday"
msgstr "Gestern"

#: ../src/gnome_abrt/tools.py:35
msgid "week"
msgstr "Woche"

#: ../src/gnome_abrt/tools.py:38
msgid "month"
msgstr "Monat"

#: ../src/gnome_abrt/tools.py:41
msgid "year"
msgstr "Jahr"

#: ../src/gnome_abrt/tools.py:44
msgid "Last {0!s}"
msgstr "Letzte(s,n) {0!s}"

#: ../src/gnome_abrt/tools.py:46
msgid "{0:d} {1!s}s ago"
msgstr "vor {0:d} {1!s}s"

#: ../src/gnome_abrt/views.py:61 ../src/gnome_abrt/views.py:207
#: ../src/gnome_abrt/views.py:209
msgid "N/A"
msgstr "N/A"

#: ../src/gnome_abrt/views.py:207
msgid " crashed"
msgstr "abgestürzt"

#: ../src/gnome_abrt/views.py:218
msgid "yes"
msgstr "ja"

#: ../src/gnome_abrt/views.py:232
msgid "no"
msgstr "nein"
